tests/functional/schema_tests/test_schema_v2_tests.py

```diff
@@ -658,6 +661,60 @@ def test_collision_test_names_get_hash(
         assert test_results[1].node.unique_id in expected_unique_ids
 
 
+class TestGenericTestsCollide:
+    @pytest.fixture(scope="class")
+    def models(self, dupe_tests_collide):  # noqa: F811
```

Comment:
[nit] `# noqa: F811` is a little unexpected here since that's usually for import statements. Does it work without this comment?

Reply:
I copied this from the other test definitions, it looks like this is a `flake8` complaint:
```
tests/functional/schema_tests/test_schema_v2_tests.py:666:22: F811 redefinition of unused 'dupe_tests_collide' from line 6
```
Indeed because `dupe_tests_collide` is being imported at the top, from the separate file of fixtures (`tests.functional.schema_tests.fixtures`)

Reply:
interesting. that seems like a flake8 bug counting a reference as a redefinition. thanks for double checking!